src/object/trampoline.cpp

```diff
@@ -85,13 +85,15 @@ Trampoline::collision(GameObject& other, const CollisionHit& hit)
     auto player = dynamic_cast<Player*> (&other);
     //Trampoline works for player
     if (player) {
+      if (player->m_does_buttjump)
+        player->m_does_buttjump = false;
```

Comment:
TODO: Check the pros and cons of this

Reply:
This is workaround for this: [video](https://cdn.discordapp.com/attachments/396022213100437504/859345418441850910/2021-06-29_10-09-15.mp4). `m_does_buttjump` is set back to true later anyway.

Reply:
Ah, okay. In this case I retract my comment.